native/nasm/Makefile

```diff
@@ -0,0 +1,17 @@
+PKG_NAME = nasm
+PKG_VERS = 2.12.01
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://www.nasm.us/pub/nasm/releasebuilds/$(PKG_VERS)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS =
+
+HOMEPAGE = http://www.nasm.us/
+COMMENT  = NASM
```

Comment:
`The Netwide Assembler, NASM, is an 80x86 and x86-64 assembler designed for portability and modularity.`, or something to that effect, might be better. Found [here](http://www.nasm.us/xdoc/2.12.01/html/nasmdoc1.html#section-1.1)